src/Microsoft.SqlTools.ServiceLayer/Migration/Contracts/GetSkuRecommendationsRequest.cs

```diff
@@ -96,4 +96,42 @@ public static readonly
             RequestType<GetSkuRecommendationsParams, GetSkuRecommendationsResult> Type =
                 RequestType<GetSkuRecommendationsParams, GetSkuRecommendationsResult>.Create("migration/getskurecommendations");
     }
+    public class SaveSkuRecommendationsParams
+    {
+        /// <summary>
+        /// List of SQL DB recommendation results, if applicable
+        /// </summary>
+        public List<SkuRecommendationResult> SqlDbRecommendationResults { get; set; }
+
+        /// <summary>
+        /// List of SQL MI recommendation results, if applicable
+        /// </summary>
+        public List<SkuRecommendationResult> SqlMiRecommendationResults { get; set; }
+
+        /// <summary>
+        /// List of SQL VM recommendation results, if applicable
+        /// </summary>
+        public List<SkuRecommendationResult> SqlVmRecommendationResults { get; set; }
+
+        /// <summary>
+        /// SQL instance requirements, representing an aggregated view of the performance requirements of the source instance
+        /// </summary>
+        public SqlInstanceRequirements InstanceRequirements { get; set; }
+    }
+
+    public class SaveSkuRecommendationsResult
+    {
+        /// <summary>
+        /// Full file names where SKU Recommendation result is saved
+        /// </summary>
+        public string[] SkuRecommendationFiles { get; set; }
```

Comment:
maybe rename to `SkuRecommendationReportFileNames` -- also, not sure if there's any difference between using `string[]` and `List<string>`, but seems most of the other contracts use List so maybe use that to be consistent?